### ISSUE TITLE
Distribute via PyPi

### ISSUE THREAD

**Issue by AlberLC** [lfhohmann/google-weather-scraper#1]
Hello! I love how well this project works.

It would be quite useful if it existed as a pip-installable package, so people could easily install and upgrade versions.

I know the process to distribute packages, if you want I can help.

**Comment by lfhohmann** [comment]
Sure, that would be awesome. How do you suggest we start it?

**Comment by AlberLC** [comment]
I will do a pull request with the new configuration files for creating a https://pypi.org/ project. You need a pypi account in order of upload projects. 

I filled all fields with your data except for version number. You must choose version every time you upload a project version.
![image](https://user-images.githubusercontent.com/37489786/156219818-4b2d9e77-99f3-4b87-9ad3-94c967fd2103.png)

Here is the tutorial https://packaging.python.org/en/latest/tutorials/packaging-projects/

**In summary:**
build and twine packages are needed to upload packages: `pip install build twine`

Then every time you want to upload a new version of google-weather-scraper you must do in project root:
`python3 -m build`
`twine upload dist/*`
and input your pypi user and password.

After that anyone can do `pip install google-weather-scraper` and install your project.
